Sources/Plasma/PubUtilLib/plAvatar/plArmatureMod.cpp

```diff
@@ -1313,7 +1313,7 @@ bool plArmatureMod::MsgReceive(plMessage* msg)
     }
 
     plLoadClothingMsg *clothingMsg = plLoadClothingMsg::ConvertNoRef(msg);
-    if (clothingMsg)
+    if (clothingMsg && fClothingOutfit)
```

Comment:
This change will cause the `plLoadClothingMsg` to be propagated down the chain of `MsgReceive()` overloads. Is this intended?

Reply:
I think that's _probably_ fine, but I'll change it to eat the message just to be safe